### PR TITLE
Adds a small help link to the toolbar

### LINKED ISSUE #2564: Documentation button in the toolbar
## Abstract

Add a "**?**" button on the toolbar taking to the readthedocs documentation.

## Motivation

The documentation on readthedocs it's really well done and should be more easy to reach for a better UX.

## Specification

Could be in the account menu or a button next to the notification bell.

### PR DESCRIPTION
**Pull request by kelsos**
Closes #2564

### CODE REVIEW

File: frontend/app/src/locales/en.json [hunk 1]
```diff
@@ -2215,6 +2215,7 @@
     "message": "rotki accounts are encrypted using your password and saved in your local filesystem."
   },
   "app": {
+    "help": "Click for help",
```
Comment (LefterisJP) [inline frontend/app/src/locales/en.json:2218]:
```suggestion
    "help": "Read the documentation",
```

**Review by LefterisJP — COMMENTED** [review]
So looks like this: 


I don't have a very strong opinion but adding something on the top right only for this isn't it a bit too much? Wouldn't it be better inside the "user settings" between settings and toggle privacy mode?

I am not sure. How visible should this be? On the one hand it's important to easily know how to visit the docs, but to use up a "slot" of the top right of the app just for a constant link feels overkill.

@isidorosp what do you think?

File: frontend/app/src/locales/en.json (hunk 1 above)
Comment (kelsos) [reply frontend/app/src/locales/en.json:2218]: Maybe just keep it more simple as `Help`

**Comment by kelsos** [comment]
> So looks like this:
> 
> 
> I don't have a very strong opinion but adding something on the top right only for this isn't it a bit too much? Wouldn't it be better inside the "user settings" between settings and toggle privacy mode?
> 
> I am not sure. How visible should this be? On the one hand it's important to easily know how to visit the docs, but to use up a "slot" of the top right of the app just for a constant link feels overkill.
> 
> @isidorosp what do you think?



> So looks like this:
> 
> 
> I don't have a very strong opinion but adding something on the top right only for this isn't it a bit too much? Wouldn't it be better inside the "user settings" between settings and toggle privacy mode?
> 
> I am not sure. How visible should this be? On the one hand it's important to easily know how to visit the docs, but to use up a "slot" of the top right of the app just for a constant link feels overkill.
> 
> @isidorosp what do you think?

If we want easy discoverability then why is it overkill? I don't have many easily accessible examples of something similar, but at least from the open windows I have I can see that discord does exactly the same but with a normally sized icon.

File: frontend/app/src/locales/en.json (hunk 1 above)
Comment (LefterisJP) [reply frontend/app/src/locales/en.json:2218]: I just am not sure about the help world alone. Made me think of some kind of support forum. But sure if you want just put help.

**Comment by LefterisJP** [comment]
Thought of something else also. Another place where we end up sending people **very** often is discord. Perhaps that could also be worthy of a link in the app. But then where does it go?  How do we differentiate all these?

**Comment by kelsos** [comment]
I saw somewhere that where they put the social media/communication links above the version 
image I am not sure though maybe we should also improve the whole user guide/

File: frontend/app/src/locales/en.json (hunk 1 above)
Comment (isidorosp) [reply frontend/app/src/locales/en.json:2218]: I'm not super crazy about putting this on the right-hand side, and for sure not in the right-most (most important/prominent) position. I also think that if something goes on the bar it should keep the same size/look/feel as the rest.

How would it look/feel if we put it on the left (after the "save status" icon?)? We could eventually maybe flesh out that area and allow a user to pick a few "shortcuts" or quick-links that they could define in their settings (i.e. open specific pages+sub-pages).

**Comment by isidorosp** [comment]
> I saw somewhere that where they put the social media/communication links above the version
> image
> I am not sure though maybe we should also improve the whole user guide and add the links there?

I kinda like this idea. Maybe we just add a "Help & Documentation" link there?

For the "socials" links I'm not sure it's worth putting buttons on the "face" of the UI (i.e. things that a person would only click once that later end up taking up space for no reason). I recall at some point you wanted to create an "about" dialog (probably under the help menu?), what if we put them there?
image

File: frontend/app/src/locales/en.json (hunk 1 above)
Comment (kelsos) [reply frontend/app/src/locales/en.json:2218]: Contextually it doesn't make sense to add it there since the rest are status icons

**Comment by kelsos** [comment]
I am closing the PR until we decide how we would like to proceed with this. Let's take the discussion on the issue and figure out what we would want to do with this.

**Pull request closed without merging**